models/spring-ai-zhipuai/src/test/java/org/springframework/ai/zhipuai/embedding/EmbeddingIT.java

```diff
@@ -53,6 +56,25 @@ void defaultEmbedding() {
 		assertThat(this.embeddingModel.dimensions()).isEqualTo(1024);
 	}
 
+	@Test
+	void embedding3() {
+		assertThat(this.embeddingModel).isNotNull();
+
+		var options = ZhiPuAiEmbeddingOptions.builder()
+			.withModel(ZhiPuAiApi.EmbeddingModel.Embedding_3.getValue())
+			.build();
+
+		EmbeddingRequest embeddingRequest = new EmbeddingRequest(List.of("Here comes the sun"), options);
+		EmbeddingResponse embeddingResponse = this.embeddingModel.call(embeddingRequest);
+
+		assertThat(embeddingResponse.getResults()).hasSize(1);
+
+		assertThat(embeddingResponse.getResults().get(0)).isNotNull();
+		assertThat(embeddingResponse.getResults().get(0).getOutput()).hasSize(2048);
+
+		// assertThat(this.embeddingModel.dimensions()).isEqualTo(2048);
```

Comment:
The changes look good! It is weird that the embedding model dimensions is "-1". Do you know why?